### PR TITLE
NEWS: News update before 1.8.1 RC1

### PR DESCRIPTION
**Pull request by shamisp**
Signed-off-by: Pavel Shamis (Pasha) <pasharesearch@gmail.com>

## What
News update before 1.8.1 rc1

## Why ?
Bugfix release

### CODE REVIEW

**Comment by shamisp** [comment]
@yosefe @tonycurtis please take a look

**Comment by shamisp** [comment]
ok to test

File: NEWS [hunk 1]
```diff
@@ -7,6 +7,15 @@
 ##
 #
 
+## 1.8.1-RC1 (June 23, 2020)
+### Bugfixes:
+- Multiple fixes in testing environment and Azure CI
```
Comment (yosefe) [inline NEWS:12]: Added binary release pipeline in Azure CI

File: NEWS [hunk 2]
```diff
@@ -7,6 +7,15 @@
...
+- Fixes in InfiniBand DEVX transport
+- Fixes in memory management for CUDA IPC transport
+- Fixes for  binutils 2.34+
```
Comment (yosefe) [inline NEWS:15]: extra space after 'for'

File: NEWS [hunk 3]
```diff
@@ -7,6 +7,15 @@
 ##
 #
 
+## 1.8.1-RC1 (June 23, 2020)
```
Comment (yosefe) [inline NEWS:10]: also need to bum version in confugure.ac to 1.8.1

Comment (shamisp) [reply NEWS:10]: Good point. Any other places that I have missed

Comment (yosefe) [reply NEWS:10]: no, this is only place, doc are generated automatically

**Comment by shamisp** [comment]
@yosefe addressed

**Pull request merged**